stable/home-assistant/templates/deployment.yaml

```
@@ -42,7 +42,7 @@ spec:
         - mountPath: {{ .Values.git.keyPath }}/id_rsa
           name: git-secret
           subPath: id_rsa
-        - mountPath: {{ .Values.git.keyPath }}id_rsa.pub
+        - mountPath: {{ .Values.git.keyPath }}/id_rsa.pub
```

To avoid this being a breaking change (for those who have already added a trailing `/` in their values file), I suggest trimming trailing `/` in the template.

```suggestion
        - mountPath: {{ .Values.git.keyPath | trimSuffix "/" }}/id_rsa.pub
```